.github/workflows/update_robottelo_image.yml

```diff
@@ -0,0 +1,89 @@
+name: update_robottelo_image
+
+on: [push]
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
+    env:
+      PYCURL_SSL_LIBRARY: gnutls
+      SATELLITE_VERSION: 6.9
+    steps:
+      - name: Checkout Robottelo
+        uses: actions/checkout@v2
+
+      - name: Set Up Python-${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get install -y libgnutls28-dev libcurl4-openssl-dev libssl-dev
+          wget https://raw.githubusercontent.com/SatelliteQE/broker/master/broker_settings.yaml.example
+          pip install -U --no-binary=pycurl -r requirements.txt -r requirements-optional.txt
+          cp robottelo.properties.sample robottelo.properties
+          cp broker_settings.yaml.example broker_settings.yaml
+          cp virtwho.properties.sample virtwho.properties
+
+      - name: Pre Commit Checks
+        uses: pre-commit/action@v2.0.0
+
+      - name: Collect Tests
+        run: |
+          pytest -n 8 --setup-plan --disable-pytest-warnings tests/foreman/ tests/robottelo/
```

Comment:
Two things that aren't blockers but I think merit discussion.

1. I would not disable pytest warnings.  Currently we see many of those, but I'm working on fixing that, #8163  is part of such effort.  We should not be getting any pytest warnings, but people would have to come here to see them.

2. The output of the workflow is a bit difficult to parse with all of the output being in one expandable block.  Setup-plan is great for debugging when working with pytest and its fixtures, but I think the output is needlessly verbose here. Collection alone satisfies what we want to test and provides a clear list of everything collected.

Reply:
@JacobCallahan @lpramuk @jyejare any thoughts?